File: JavaScript/프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.js
function solution(num_list) {
    var answer = [];
    let e = 0, o = 0;
    
    for(let i=0; i<num_list.length; i++) {
        if(num_list[i] % 2 == 0) 
            e++;
        else 
            o++;
    }
    answer = [e, o];
    return answer;
}

File: Python3/프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.py
def solution(num_list):
    answer = []
    e = 0
    o = 0
    for i in range(len(num_list)):
        if num_list[i] % 2 == 0:
            e += 1
        else:
            o += 1
    answer.append(e)
    answer.append(o)
    return answer